### PR TITLE
[#164097940] - Covering the tests for all endpoint.

### PR DESCRIPTION
**Pull request by niyobobo**
#### What does this PR do?
* Implementation of test-driven development to all endpoint

#### Description of Task to be completed?
* Tests implementation for the newly created endpoints (Auth and election endpoint)
* Added token authentication for the remained features and perform tests with authentication enabled
* Fixed Travis CI issues of failing test (configure environment variable in Travis)

#### How should this be manually tested?
They can be manually viewed through server/tests directory.

#### Any background context you want to provide?
Tests used for development purpose to check if all features implemented as expected.

#### What are the relevant pivotal tracker stories?
[#164097940 - Developer should be able to cover the tests of all endpoint.)](https://www.pivotaltracker.com/n/projects/2240018)

### CODE REVIEW

File: server/database/connection.js [hunk 1]
```diff
@@ -1,12 +1,14 @@
 const { Pool } = require('pg');
```
Comment (FredDominant) [inline server/database/connection.js]: change your `require` to `import`

**Review by FredDominant — CHANGES_REQUESTED** [review]
Address the comments then merge.

File: server/database/connection.js (hunk 1 above)
Comment (niyobobo) [reply server/database/connection.js]: I fixed this,
Converted connection.js file to ES6

**Review by FredDominant — COMMENTED** [review]
Approved

**Pull request merged**